dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -1381,35 +1343,6 @@ export default defineComponent({
         index
```

Comment:
Question: In line 1328 your write:
`this.listOfProductionSites[index].listOfGoodsOrServices = [
        ...this.listOfProductionSites[index].listOfGoodsOrServices,
        ...items,
      ];`

Can't you just push the `items` to the existing `this.listOfProductionSites[index].listOfGoodsOrServices` ?
Why do you have to re-assign the old values + the items?

Reply:
And why is the `const` called `items`?
Isn't it actually a list of goods or services?  So maybe `const listOfGoodsOrServicesToAdd` ?

Reply:
When I change it directly vue didn't reload the view. That's why there is this reassignment.
I changed name to listOfGoodsOrServicesToAdd

Reply:
In the internet I have read, that Vue reloads the view if you do the push inside of a method.

E.g.
`addItemsToListOfGoodsOrServices(indexOfProductionSite: number, items: string[] ) {
   this.listOfProductionSites[index].listOfGoodsOrServices.push(items)
}

We will try this and check if it works. You don't have to do it :)

Reply:
It worked with "push" for me. Vue is reloading the view. Therefore I changed it to "push".